mis/pipeline/basesolver.py

```diff
@@ -31,14 +31,19 @@ def __init__(self, instance: MISInstance, config: SolverConfig):
             instance (MISInstance): The MISproblem to solve.
             config (SolverConfig): Configuration settings for the solver.
         """
-        self.instance: MISInstance = instance
+        self.original_instance: MISInstance = instance
```

Comment:
is it worth to `deepcopy()` the instance at some point? Or is it done elsewhere?